get_endpoints.js

```diff
@@ -0,0 +1,102 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * Function for getting AL enpoints. 
```

Comment:
this module is not required.
There is `EndpointsC` already available in `al-collector-js`.
Please see https://github.com/alertlogic/al-azure-collector-js/blob/master/master.js#L98 as an example.